x/leverage/client/proposal_handler.go

```diff
@@ -1,27 +1,14 @@
 package client
 
 import (
-	"net/http"
-
-	"github.com/cosmos/cosmos-sdk/client"
-	"github.com/cosmos/cosmos-sdk/types/rest"
 	govclient "github.com/cosmos/cosmos-sdk/x/gov/client"
-	govrest "github.com/cosmos/cosmos-sdk/x/gov/client/rest"
 
 	"github.com/umee-network/umee/v2/x/leverage/client/cli"
 )
 
+// TODO: need to be updated to new gov message hanlders
+//    https://github.com/umee-network/umee/issues/1001
```

Comment:
```suggestion
// TODO #1001 : update to use new gov message handlers
```

Reply:
Do we use that style often in other places?

Reply:
When we have a linked issue in the same repo, yeah. If you search for `TODO` in the main branch (using find and replace for example) you'll find 

- 8 uses of `TODO #` in leverage
- 17 (irrelevant) uses of `context.TODO()`
- 3 uses of `TODO` with a full link on the next line
- 1 use of `TODO` without a link

It looks like I'm the only one using the short format though, so feel free to ignore.

Reply:
I don't have a strong preference, but full link makes it easier to navigate (don't need to glue the link when opening it from an editor). It's just a small detail.